composer/trainer/trainer.py

```diff
@@ -304,6 +307,34 @@ def _is_tpu_installed() -> bool:
     import torch_xla.distributed.parallel_loader as pl
 
 
+def _create_object_store_logger_from_uri(uri: str) -> Optional[LoggerDestination]:
+    parse_result = urlparse(uri)
+    backend, bucket = parse_result.scheme, parse_result.netloc
+    if backend == 's3':
+        return ObjectStoreLogger(object_store_cls=S3ObjectStore,
+                                 object_store_kwargs = {'bucket': bucket})
+    elif backend == 'wandb':
+        return WandBLogger(log_artifacts=True)
+    elif backend == '': # Local path
+        return None
+    else:
+        raise NotImplementedError(f'There is not implementation for the cloud backend {backend}. Please use either'
```

Comment:
"There is not implementation" -> "There is not an implementation"

Reply:
fixed